### PR TITLE
[ixwebsocket] Update to 8.0.5

### PR DESCRIPTION
**Pull request by bsergean**
**Describe the pull request**

- PR update a library to 8.0.5
- Platforms supported are mac, linux, windows, and uwp / I'm not super sure about those platforms.

### CODE REVIEW

**Comment by NancyLi1013** [comment]
Hi @bsergean 

Thanks for this PR.
Could you please resolve the conflicts against master first?

In the meanwhile, I noticed there are some features listed in `CONTROL `file.
But I didn't see any usage about these features in `portfile.cmake`.
Are these features not supported currently?

**Comment by bsergean** [comment]
Hi Nancy,

I rebased yesterday, I think it should be good now.

The CONTROL file is all about setting up the right build flags depending on the platform.

* Windows can only build with TLS support with mbedlts as the backend now. (although someone managed to get it to work with OpenSSL I think)
* Linux can use openssl or mbedlts, but we still default it to openssl
* apple uses the default apple TLS stack.

**Review by NancyLi1013 — APPROVED** [review]

File: ports/ixwebsocket/CONTROL [hunk 1]
```diff
@@ -1,5 +1,4 @@
-Source: ixwebsocket
-Version: 7.9.2
+Version: 8.0.5
```
Comment (strega-nil) [inline ports/ixwebsocket/CONTROL]: Why was the `Source` field removed?

**Comment by strega-nil** [comment]
Alright, looks good to me! Thanks @bsergean :)

**Comment by bsergean** [comment]
Thanks !

**Pull request merged**